analyzers/pylint.py

```diff
@@ -25,61 +24,62 @@ class PylintAnalyzer:
     @staticmethod
     def define_arguments():
         parser = argparse.ArgumentParser(description="Pylint analyzer")
-        parser.add_argument("--files", dest="file_list", nargs='+', help="Python files and Python packages for Pylint.")
-        parser.add_argument("--rcfile", dest="rcfile", help="Specify a configuration file.")
+        parser.add_argument("--files", dest="file_list", nargs='+', required=True,
+                            help="Python files and Python packages for Pylint. " + \
+                                 "Files could be defined as a single python file" + \
+                                 " *.py or directories with __init__.py file in the directory.")
+        parser.add_argument("--rcfile", dest="rcfile", type=str, help="Specify a configuration file.")
         parser.add_argument("--python-version", dest="version", default="3", choices=["2", "3"],
                             help="Version of Python")
         utils.add_common_arguments(parser)
         return parser
 
     def __init__(self, settings):
         self.settings = settings
-        self.json_file = settings.result_file
 
     def execute(self):
-        if not self.settings.file_list:
```

Comment:
Why you dropped this check?

Reply:
because I add 'required=True' to parameter